### DISCUSSION TITLE
How to share tmpfs on /dev/shm?

### DISCUSSION THREAD

**Discussion by superiums** [netblue30/firejail#5471] [Q&A] [answered]
i always let browser to download files to tmpfs (/dev/shm) to reduce io.
but firejails always mount another tmpfs on /dev/shm for browser, differ from the one on host. 

so how could i sharing them ?

**Comment by rusty-snake** [comment, marked as answer]
1. Open the manpage (`man 1 firejail`).
2. Search for `/dev/shm`.
3. Find **`--keep-dev-shm`** with description "`/dev/shm` directory is untouched (even with `--private-dev`)".

---

BTW: Using a subdir of `/tmp` might be better than `/dev/shm`.

**Reply by superiums** [reply]
thank you very much!
